Makefile

```diff
@@ -343,6 +343,20 @@ darwin-amd64: require-assets
 		echo "-> Skipped: Can not compile Lantern for OSX on a non-OSX host."; \
 	fi
 
+lantern: require-assets
+	@echo "Building development lantern" && \
+	export OSX_DEV_SDK=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX$(OSX_MIN_VERSION).sdk && \
+	if [[ "$$(uname -s)" == "Darwin" ]]; then \
+		source setenv.bash && \
+		$(call build-tags) && \
+		if [[ -d $$OSX_DEV_SDK ]]; then \
+			export CGO_CFLAGS="--sysroot $$OSX_DEV_SDK" && \
+			export CGO_LDFLAGS="--sysroot $$OSX_DEV_SDK"; \
+		fi \
+	fi
+	MACOSX_DEPLOYMENT_TARGET=$(OSX_MIN_VERSION) \
+	CGO_ENABLED=1 GOOS=darwin GOARCH=amd64 go build -race -o lantern -tags="$$BUILD_TAGS" -ldflags="$(LDFLAGS) $$EXTRA_LDFLAGS -s" github.com/getlantern/flashlight/main; \
```

Comment:
For this I'd be tempted to skip all the custom SDK stuff and just use whatever go you've got on your path. You just trying to make it consistent with real builds @oxtoacart?